rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/helm/HelmBakeManifestRequest.java

```diff
@@ -23,6 +23,13 @@ public class HelmBakeManifestRequest extends BakeManifestRequest {
 
   boolean rawOverrides;
 
+  /**
+   * Helm v3 adds a new flag to include custom resource definition manifests in the templated
+   * output. In the previous versions crds were usually included as part of templates, so the `helm
+   * template` command always included them in the rendered output.
+   */
+  boolean includeCRDs;
```

Comment:
Is this needed?

Reply:
> Is this needed?

otherwise it does not compile line 112 of HelmTemplateUtils